cpp/ps0005/Task5.cpp

```diff
@@ -17,59 +18,62 @@ int main()
     cout << "Enter the operator / * - + % : \n";
     cin >> r;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input